src/static/js/add_tta_match_result.js

```diff
@@ -54,17 +53,25 @@ function handleTournamentsResponse(responseText) {
   initializePage();
 }
 
+function getTournamentNameFromURL() {
+  // Expected URL is "http://..../tournament/<name>/add_ffa_match_result"
```

Comment:
nit: fix comment to be tta instead of ffa XD

Reply:
done